longdesc_label.lua

```diff
@@ -24,12 +25,13 @@ function chaosforge.longdesc_label:show()
     local message = amap.localization.current_long
     local window_width = getMainWindowSize()
     local label_width = window_width
-    local chars_per_line = math.floor(label_width / 10)
+    local chars_per_line = math.floor(label_width / 12)

```

Comment:
local chars_per_line = math.floor(getMainWindowSize() / 12) ??

Reply:
wyższa kombinatoryka znak zajmuje srednio 12px

Reply:
nie nie, chodzi mi o to ze deklarujesz 2 zmienne od linii 25 do 26, gdzie moglbys ppoprostu uzyc getMainWindowSize() bezposrednio

Reply:
nie wnikam czy tam jest 10 czy 12 :)